backend/tracim_backend/radicale_plugins/radicale_none_right/__init__.py

```diff
@@ -0,0 +1,19 @@
+from radicale import pathutils
+from radicale import rights
+
+
+class Rights(rights.BaseRights):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._verify_user = self.configuration.get("auth", "type") != "none"
+
+    def authorization(self, user, path):
+        if self._verify_user and not user:
+            return ""
+        sane_path = pathutils.strip_path(path)
+        if "/" not in sane_path:
+            return "RW"
```

Comment:
Is is logic that `RW` is returned here and `rw` two lines later?

Reply:
Yes the meaning is not the same: https://radicale.org/v3.html#authentication-and-rights
RW -> read/right collection
rw -> read/right calendar/addressbook.